### PR TITLE
Add guide for stability-monitor

### PR DESCRIPTION
**Pull request by mmoscicki2**
### Change Description ###

Added porting guide for stability-monitor.

Continuation of https://github.com/Samsung/tizen-docs/pull/1417

### CODE REVIEW

File: docs/platform/porting/system.md [hunk 1]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+### Stability monitor
+
+The purpose of the stability monitor is to keep track of the usage of system resources with low overhead. The resources that are tracked for maximum and average usage are as follows:
+-  CPU
+-  Memory
+-  Input/Output
+-  Open file descriptors (only maximum value is measured, not average).
+
+Resources are consumed by various processes and are classified according to their type: application or service. Applications have the type as native, web app and .NET, with the state as foreground and background. The process parameters are distinguished in configuration, and various parameters can be assigned to different services and applications. When excessive use of resources abnormality is detected, an appropriate D-Bus signal is sent, and an optional abnormality report is generated.
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2552]:
```suggestion
Resources are consumed by various processes and are classified according to their type: application or service. Applications have the type as native, web app, and .NET, with the state as foreground and background. The process parameters are distinguished in configuration and various parameters can be assigned to different services and applications. When excessive use of resources abnormality is detected, an appropriate D-Bus signal is sent, and an optional abnormality report is generated.

File: docs/platform/porting/system.md [hunk 2]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+#### Kernel module
+
+The stability monitor relies on the kernel module, which collects information about processes state in the system in most resource efficient way. This raw data is exposed via `/proc/tsm` interface.
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2556]:
```suggestion
The stability monitor relies on the kernel module, which collects information about processes state in the system in the most resource-efficient way. This raw data is exposed using the `/proc/tsm` interface.

File: docs/platform/porting/system.md [hunk 3]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+The following resource statistics are collected:
+-  CPU: system time and user time [cpu_usage = d(system_time + user_time) / dt].
+-  Memory: RSS.
+-  I/O: bytes read and bytes written.
+-  FDs: number of open file descriptors.
+
+#### Service
+
+The purpose of stability-monitor service is to:
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2566]:
```suggestion
The purpose of stability monitor service is to:

File: docs/platform/porting/system.md [hunk 4]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+     Default value: `4`
+
+-  **mem_limit_peak**  
+     Specifies the maximum allowed memory usage. A value of `1.0` means that the process can use all available memory. It is system-wide parameter, and there is no way to specify it in megabytes.
+
+     Default value: `1`
+
+-  **io_limit_peak**  
+     Specifies the maximum I/O usage in megabytes per second (MB/s). The I/O usage is unlike the usage of CPU and memory values.
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2661]:
```suggestion
     Specifies the maximum I/O usage in Megabytes per second (MB/s). The I/O usage is unlike the usage of CPU and memory values.

File: docs/platform/porting/system.md [hunk 5]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+     Default value: `1`
+
+-  **cpu_limit_peak**  
+     Specifies the maximum allowed CPU usage. A value of `1.0` means that the process can consume the CPU time equivalent to using one core at 100%.
+
+     Default value: `4`
+
+-  **mem_limit_peak**  
+     Specifies the maximum allowed memory usage. A value of `1.0` means that the process can use all available memory. It is system-wide parameter, and there is no way to specify it in megabytes.
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2656]:
```suggestion
     Specifies the maximum allowed memory usage. A value of `1.0` means that the process can use all available memory. It is the system-wide parameter, and there is no way to specify it in Megabytes.

File: docs/platform/porting/system.md [hunk 6]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+     Default value: `0`
+
+-  **report**  
+     Defines whether to create bug report after detecting abnormality.
+
+     Default value: `0`
+
+-  **sampling_rate**  
+     Specifies the sampling rate (Hz). This is the process data update frequency. Maximum allowed value is specified by **sampling_rate** from the `global` section.
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2646]: Shouldn't **sampling_rate** be in code styling instead? If no, ignore the comment.

File: docs/platform/porting/system.md [hunk 7]
```diff
@@ -2540,3 +2540,353 @@ Example:
...
+
+    "9zWvGSYU8Z.calendar": {
+        "print_current": 1,
+    }
+}
+```
+
+#### D-Bus signal
+
+When the abnormality is detected, the stability monitor sends one of the following D-Bus signals, depending on the module or parameter that detected or caused the abnormality:
```
Comment (nawab-samsung) [inline docs/platform/porting/system.md:2792]:
```suggestion
When the abnormality is detected, the stability monitor sends one of the following D-Bus signals, depending on the module or parameter that detects or causes the abnormality:

**Review by nawab-samsung — CHANGES_REQUESTED** [review]
@mmoscicki2 Giving the last set of comments, just to ensure consistency within the page. Please incorporate the comments. 
Thanks.

**Review by nawab-samsung — APPROVED** [review]
@mmoscicki2 Thanks for incorporating the comments.

**Comment by mmoscicki2** [comment]
@nawab-samsung Thank you for review

**Pull request merged**